### PR TITLE
Make stale issues/PR checker into scripts

### PR DESCRIPTION
**Pull request by astrofrog**
@Cadair - is this what you had in mind?

```
$ check-stale-issues -h
usage: check-stale-issues [-h] --repository REPOSITORY --warn-seconds WARN_SECONDS
                          --close-seconds CLOSE_SECONDS

Check for stale issues and close them if needed.

optional arguments:
  -h, --help            show this help message and exit
  --repository REPOSITORY
                        The repository in which to check for stale issues
  --warn-seconds WARN_SECONDS
                        After how many seconds to warn about stale issues
  --close-seconds CLOSE_SECONDS
                        After how many seconds to close stale issues
```
```
$ check-stale-pull-requests -h
usage: check-stale-pull-requests [-h] --repository REPOSITORY --warn-seconds WARN_SECONDS
                                 --close-seconds CLOSE_SECONDS

Check for stale pull requests and close them if needed.

optional arguments:
  -h, --help            show this help message and exit
  --repository REPOSITORY
                        The repository in which to check for stale pull
                        requests
  --warn-seconds WARN_SECONDS
                        After how many seconds to warn about stale issues
  --close-seconds CLOSE_SECONDS
                        After how many seconds to close stale issues
```

### CODE REVIEW

File: baldrick/scripts/stale_pull_requests.py [hunk 1]
```diff
@@ -103,33 +88,65 @@ def process_pull_requests(repository, installation):
...
+    parser.add_argument('--close-seconds', dest='close_seconds', action='store',
+                        type=int, required=True,
+                        help='After how many seconds to close stale issues')
+
+    parser.add_argument('--bot-name', dest='bot_name', required=False,
+                        help='The name of the bot (used if a Flask application doe not already exist)')
+
+    args = parser.parse_args(argv or sys.argv[1:])
+
+    try:
```
Comment (Cadair) [inline baldrick/scripts/stale_pull_requests.py]: is this *ever* going to work?! lol

**Comment by Cadair** [comment]
Its looking nice, I do wonder if we could refactor this to be completely separate from the app? Just use env vars, command line args, and master for config?

**Comment by astrofrog** [comment]
Done - no Flask app is required for this to work now!

File: baldrick/github/github_auth.py [hunk 2]
```diff
@@ -112,7 +112,7 @@ def github_request_headers(installation):
     return headers
 
 
-def repo_to_installationid_mapping():
+def repo_to_installation_id_mapping():
```
Comment (Cadair) [inline baldrick/github/github_auth.py]: Just a general point, this function is *very* expensive in time and API calls to GitHub we should think about trying to cache it. However, we need to be careful about how we invalidate that cache.

**Pull request merged**